org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WRecordInfo.java

```diff
@@ -235,8 +231,10 @@ private void init (boolean showTable) throws Exception
 		south.setSclass("dialog-footer");
 		south.setParent(layout);
 		//
+		m_permalink.setStyle("text-decoration:underline; color:#0093F9;");

```

Comment:
maybe this style is better to apply one from the css theme?

also this could be shown maybe as a button instead of a link?